cg/models/slurm/sbatch.py

```diff
@@ -1,16 +1,28 @@
 from typing import Optional
 
 from pydantic import BaseModel
-from typing_extensions import Literal
 
 from cg.constants.priority import SlurmQos
 
 
 class Sbatch(BaseModel):
     job_name: str
     account: str
+    log_dir: str
```

Comment:
those classes are pretty similar!
Could something like this be possible:?
```python
class SbatchDragen(Sbatch):
    partition: str = "dragen"
    nodes: int = 1
    cpus_per_task: int = 24

```

Reply:
fixed: https://github.com/Clinical-Genomics/cg/pull/1204/commits/3b0c5b548678a45d159285e121cc4c91f56c64ab